bitnami/dataplatform-bp1/Chart.yaml

```diff
@@ -0,0 +1,52 @@
+annotations:
```

Comment:
Could you execute this:
```
yq eval --tojson Chart.yaml | jq -S . | yq  eval --prettyPrint - > Chart.yaml.new
mv Chart.yaml.new Chart.yaml
```
it will sort the file